src/mame/casio/pv1000.cpp

```diff
@@ -89,40 +102,63 @@ void pv1000_sound_device::voice_w(offs_t offset, uint8_t data)
 
 
 /*
- plgDavid's audio implementation/analysis notes:
+	plgDavid's audio implementation/analysis notes:
 
- Sound appears to be 3 50/50 pulse voices made by cutting the main clock by 1024,
- then by the value of the 6bit period registers.
- This creates a surprisingly accurate pitch range.
+	Sound appears to be 3 50/50 pulse voices made by cutting the main clock by 1024,
+	then by the value of the 6bit period registers.
+	This creates a surprisingly accurate pitch range.
 
- Note: the register periods are inverted.
- */
+	Note: the register periods are inverted.
+
+	plgDavid 2023 update: lidnariq (NESDEV) took a fondness to the system and gave me a bunch of test roms 
+	to strenghten the emulation. 
+	
+	Quite a few things were uncovered overall, but for audio specifically:
+	1)Audio mix/mux control ($FB, case 0x03) was ignored
+	2)lidnariq's tracing of my PCB scans showed that all three sound outputs are mixed using different volumes:
+	  square1 via i/o$F8 is -6dB, square2 via i/o$F9 is -3dB, defining square3 via i/o$FA as 0dB
```

Comment:
The tabs in comments beyond the opening delimiter indent will be replaced with spaces by srcclean.  Please run srcclean on this file to reduce churn.

Reply:
Right. Too many different editors having different tabs/space settings.